jsonschema-validation.xml

```diff
@@ -536,84 +530,66 @@
                     <xref target="json-schema">core JSON Schema.</xref>
                     <cref>
                         Note that the "type" keyword in this specification defines an "integer" type
-                        which is not part of the data model.  Therefore a format attribute can be
-                        limited to numbers, but not specifically to integers.  However, a numeric
+                        which is not part of the data model. Therefore a format attribute can be
+                        limited to numbers, but not specifically to integers. However, a numeric
                         format can be used alongside the "type" keyword with a value of "integer",
                         or could be explicitly defined to always pass if the number is not an integer,
                         which produces essentially the same behavior as only applying to integers.
                     </cref>
                 </t>
 
                 <t>
-                    Meta-schemas that do not use "$vocabulary" SHOULD be considered to
-                    utilize this vocabulary as if its URI were present with a value of false.
-                    See the Implementation Requirements below for details.
+                    The current URI for this vocabulary, known as the Format-Annotation vocabulary, is:
+                    &lt;https://json-schema.org/draft/2020-11/vocab/format-annotation&gt;.  This vocabulary
+                    is required by this specification.
```

Comment:
```suggestion
                    is required to be implemented by this specification.
```

Reply:
That language is odd (the spec isn't doing the implementing) and a little redundant (I would think it's implied that a requirement would need to be implemented).

Reply:
Agree with @gregsdennis although the `required` should be `REQUIRED`.

Reply:
I see @karenetheridge's point, though.  I'll clean this up and make the requirements explicit for each vocabulary (`REQUIRED` for annotation, `OPTIONAL` for assertion).

Reply:
✔️